qiita_db/handlers/tests/test_processing_job.py

```diff
@@ -35,15 +36,23 @@ def test_get_job(self):
 
 
 class JobHandlerTests(TestHandlerBase):
+    def setUp(self):
```

Comment:
I've seen this construction quite a few times already - I think it is worth to move it to a base class setUp so we avoid code duplication.